desc/compute/_metric.py

```diff
@@ -1899,6 +1899,42 @@ def _gradzeta(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="g^aa",
+    label="g^{\\alpha \\alpha}",
+    units="m^{-2}",
+    units_long="None",
```

Comment:
inverse square meters

Reply:
done

Reply:
is the change in your local branch? I dont see it on git